### PR TITLE
Add missing equality on correct branch

### PR DESCRIPTION
**Pull request by Nyan11**
on geometry / tranform origin / layout / backgro…und + TU

### CODE REVIEW

**Comment by Nyan11** [comment]
See: https://github.com/pharo-graphics/Bloc/pull/468

**Comment by Nyan11** [comment]
Should fix: https://github.com/pharo-graphics/Bloc/issues/469

File: src/Bloc/BlMatrixTransformation.class.st [hunk 1]
```diff
@@ -19,6 +19,10 @@ BlMatrixTransformation class >> matrix: aBlMatrix [
 		  yourself
 ]
 
+{ #category : #comparing }
+BlMatrixTransformation >> = anObject [
+]
```
Comment (tinchodias) [inline src/Bloc/BlMatrixTransformation.class.st:24]: This returns self, not a Boolean! it should compare the `matrix` instvar, right?

Comment (tinchodias) [reply src/Bloc/BlMatrixTransformation.class.st:24]: I'm sorry I didn't see it before merging it @Nyan11

Comment (Nyan11) [reply src/Bloc/BlMatrixTransformation.class.st:24]: Oups, it should not be there. I didn't do equality for Transformation. Only for transformation origin

Comment (Nyan11) [reply src/Bloc/BlMatrixTransformation.class.st:24]: I fix it in https://github.com/pharo-graphics/Bloc/pull/473

**Pull request merged**